targets/bcm27xx/patches/50-add-eth1-to-board-config.patch

```diff
@@ -0,0 +1,10 @@
+--- a/target/linux/bcm27xx/base-files/etc/board.d/02_network	2021-08-21 22:15:47.655631601 +1000
```

Comment:
Could this be handled instead in the same way as x86 ethernet?
https://github.com/ericpaulbishop/gargoyle/blob/base_on_openwrt_2203/targets/x86/patches/50-x86setup.patch

Only a suggestion, your call. It's a neat patch either way.

Reply:
I'm sure the network change could be handled the same way as x86 and I looked at doing that when I assembled the bcm27xx target originally against 1.13.x/OWrt19.07.  I can't remember where I found the basis of the technique in this patch (most likely another OpenWrt target 02_network file I think) but I used it because it was so much simpler and worked for most bcm27xx boards (which have an onboard ethernet port, even if it's USB attached, which is reliably enumerated as eth0 before any extra USB interfaces are added) without appearing to interfere with use of LTE dongles.  It worked on 1.15.x too in my limited testing so I kept it.  Based on what I've read, not yet having been able to test, the PCI NIC on the DFRobot CM4 router board is enumerated as eth1 too so I expect that setup to work the same as the Pi4 with a USB ethernet dongle.  I don't have any of the WiFi only devices (e.g. Pi0W) to test to see what would be needed for them.

The x86 ethernet situation undoubtedly requires more sophistication/complexity; so far I don't think that's needed for bcm27xx so would prefer to keep this patch until it proves insufficient.